Makefile

```diff
@@ -307,6 +314,13 @@ endif
 dist/agent-freebsd-amd64: seego
 	$(call SetBuildVarsConditional,freebsd);  $(seego) build $(CGO_FLAGS) -o $@ ./cmd/agent
 
+#########################
+# BEGIN AGENTSMOKE DIST #
+#########################
+
+dist/agent-smoke-darwin-arm64: seego
+	$(call SetBuildVarsConditional,darwin/arm64) ;     $(seego) build $(CGO_FLAGS) -o $@ ./cmd/agent-smoke
+
```

Comment:
Do we want to support darwin x64?

Reply:
I can remove this, I had originally just added this target to test locally but we won't want to ship this.